spec/controllers/api/v0/user_roles_controller_spec.rb

```diff
@@ -26,19 +26,7 @@
       it 'fetches list of roles of a user' do
         get :index_for_user, params: { user_id: user_whose_delegate_status_changes.id }
 
-        expect(response.body).to eq([{
-          id: 'delegate-' + user_whose_delegate_status_changes.id.to_s,
-          end_date: nil,
-          is_active: true,
-          group: user_senior_delegate_role.group,
-          user: user_whose_delegate_status_changes,
-          is_lead: false,
-          metadata: {
-            status: "candidate_delegate",
-            location: "Australia",
-          },
-          class: 'userrole',
-        }].to_json)
```

Comment:
About this deletion: I see that you're removing the "manually glued together" roles in this PR. But shouldn't there be a replacement of a "clean" role object being sent instead?
I don't get why this `expect` gets deleted, from my perspective it should be _replaced_ with an equivalent "proper" role serialization. Can you please clarify?

Reply:
No, I think the way you outlined will be fine :) Thanks for the explanation!